content/crutchcorn/collections/art-of-accessibility/posts/art-of-a11y-text/index.md

```diff
@@ -0,0 +1,561 @@
+| JAWS          | List all headings                                            | <kbd>Insert</kbd> + <kbd>F6</kbd>                            |
+| JAWS          | Go to next heading of level [1-6]                            | <kbd>1</kbd> - <kbd>6</kbd>                                  |
+
+## Heading order
+
+Because headings are often used to orient users on a page, it's important that we structure them in ways that make sense at a quick "glance". This is where heading order becomes very important.
+
+Many of the rules around the order of headings come from how we naturally count. Let's explore two different rules that resonate with this natural counting method.
+
+### One `h1` per page
+
+While most programmers might disagree, ask most people to count to `10` and they'd likely start at `1`.
+
+Similarly, you want to make sure that each of your pages has an `h1` at the top of the page:
+
+```html
+<!-- This is good! -->
+<h1>Search page</h1>
+<!-- ... -->
+<h2>Collections</h2>
+<!-- ... -->
+<h2>Posts</h2>
+<!-- ... -->
+```
+
+However, while other heading levels can be repeated without any concern, **`h1`s are expected to be uniquely added only once per page**, to allow the rest of the page's structure to be defined by an `h1`:
+
+```html
+<!-- This is bad -->
+<!-- "Collections" and "Posts" shoud be h2s -->
+<h1>Search page</h1>
+<!-- ... -->
+<h1>Collections</h1>
+<!-- ... -->
+<h1>Posts</h1>
+<!-- ... -->
+```
+
+### Descending heading orders
+
+If we asked our imaginary population to count to `10` again, we'd find that most folks are likely to assume an increment of one counting up: `1`, `2`, `3`, and so on.
+
+After all, it would be atypical for someone to go from `2` to `4` when counting upwards, right?
+
+The same applies for headings: **Headings are expected to be added in a descending order of `1` level of depth at a time**.
+
+For example:
+
+```html
+<h2>Heading two</h2>
+<!-- Good! Went from 2 to 3 -->
+<h3>Heading three</h3>
+<!-- Bad! Missing an `h4` -->
+<h5>Heading three</h5>
+```
+
+However, while you should only count heading levels down by `1` at a time, you can jump back up the count by any number you'd like:
+
+``` html
+<!-- This is all allowed -->
+<h2>Heading two</h2>
+<h3>Heading three</h3>
+<h4>Heading four</h4>
+<h2>Heading five</h2>
+```
+
+## Content Grouping
+
+When working with content that has a distinct grouping of information, it may be tempting to assume that the user can simply use headings to distinguish different parts of the site.
+
+Alas, this isn't true! Take the following example:
+
+![A headphone sales marketing page with a description, features list, and customer reviews](./headphones_mockup.png)
+
+This mockup might be naïvely marked up as such:
+
+```html
+<h1>EchoFlow X1</h1>
+<img src="headphone.jpg" alt="Black headphones with 'EchoFlow X' written on the earcup.">
+<h2 class="visually-hidden">Description</h2>
```

Comment:
I feel like this as a novice's intro to `visually-hidden` styles could lead to a overuse of these sometimes necessary hacks that lead to new issues (not unlike the mantra of "the best ARIA is no ARIA").

Without this heading, I don't feel like anything is wrong. The description paragraph belongs just as much to the `<h1>` as the image. Assistive tech would also have the same experience as someone using the screen, which is ideal because these aren't mutually exclusive. Imagine a screen reader user that uses the screen, for instance, they could get frustrated by this sort of usage if common on the website.